independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus/tooling/github-action/base/.github/workflows/ci.tpl.qute.yml

```diff
@@ -12,11 +12,17 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - name: Set up JDK {java.version}
-        uses: actions/setup-java@v1
+        uses: actions/setup-java@v3
         with:
           java-version: {java.version}
+          distribution: temurin
+          {#if buildtool.cli == 'gradle'}
+          cache: gradle
+          {#else}
+          cache: maven
+          {/if}
```

Comment:
```suggestion
          cache: '{buildtool.cli}'
```

Reply:
I did that because AFAIK `buildtool.cli` is `mvn` in the case of Maven?

Reply:
Oh bummer, nvm then

Reply:
No I said mvn :]